main/lib/src/FileWriterLCIOC.cc

```diff
@@ -33,9 +33,10 @@ namespace eudaq {
     // int m_runnumber; 
   };
 
-  namespace {
-    static RegisterFileWriter<FileWriterLCIOC> reg("calice");
-  }
+  //namespace {
```

Comment:
You could probably remove those commented lines directly instead of having dead code laying around.